news_items/041_element_question_writing/index.html

```diff
@@ -0,0 +1,172 @@
+<style>
```

Comment:
Tangential thought: we should really update the news item machinery to support writing them in Markdown.

Reply:
I had the same thought while writing this 😅